generators/app/templates/app/middlewares/apiInfo.js

```diff
@@ -0,0 +1,16 @@
+const { merge } = require('lodash');
+const pjson = require('../../package.json');
+
+exports.apiInformation = (req, res, next) => {
+  try {
+    if (!res.headers) {
+      res.headers = {};
+    }
+    merge(res.headers, { 'X-API-Date': new Date() });
+    merge(res.headers, { 'X-Package-Version': pjson.version });
+    merge(res.headers, { 'X-Node-Version': pjson.engines.node });
```

Comment:
Would be nice that headers name could be configurable from env vars, with a default value to this